src/components/search/SearchField.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Card } from 'antd';
+
+import useModal from '../Modal/useModal';
+import SearchModalContent from '../Modal/SeachModalFragment';
+import data from '../search-bar/MOCK_DATA.json';
+
+const SearchField = ({ value }) => {
+  //state variables
+  const [toggle, setToggle] = useState(false);
+  const [options, setOptions] = useState([]);
+  const [list, setList] = useState([]);
+  //custom hooks
+  const { isVisible, selectedUser, toggleModal } = useModal(Modal);
+
+  const filterData = (value, options) => {
```

Comment:
I had most of this locally from the debugging we worked on monday night. decided to push them so they wouldn't need to be rewritten. I think we're in a good place to merge this. Although I'm not sure I can really review my own changes...